data/data/azure/vnet/vnet.tf

```diff
@@ -1,20 +1,24 @@
+# XXX: This isn't used anywhere
```

Comment:
Is it used for egress routing? Probably should either remove the comment or the resource.

Reply:
Removed!